component-library/src/assets/locales/en.json

```diff
@@ -685,5 +685,149 @@
+      "bannerDismissDoAlt": "A banner that has the label “You’re almost done with your profile” and description text with a close icon button. The banner uses the teal info styling.",
+      "bannerDismissDontAlt": "A banner that has the label “You’re almost done with your profile” and description text with a CTA button that says “Dismiss”. The banner uses the teal info styling.",
+      "bannerDismissDoText": "Use the close icon button to close the banner.",
+      "bannerDismissDontText": "Don’t use CTA button to close the banner.",
+      "BannerMinMaxWidthLabel": "Minimum and maximum widths",
+      "bannerMinMaxWidthAlt": "Alt text: Specs displaying the minimum and maximum widths of the banner component. Widths are described in the paragraph below.",
+      "bannerMinMaxWidthText":"Small banners have a maximum width of 600 pixels and a minimum width of 300 pixels. Large banners have a maximum width of 680 pixels and a minimum width of 340 pixels. "
+    },
+    "Anatomy" : {
+      "ImgAlt": "The parts that makeup the banner component. The parts are described in the content below.",
+      "ContainerHeading":  "Container",
+      "ContainerText":  "The boundary of the banner. The container can display various styles and can be set to any desired fixed width.",
+      "BannerIconTypeHeading":  "Banner type icon",
+      "BannerIconTypeText":  "Icon indicating the type / sentiment of the banner.",
+      "BannerBarHeading":  "Banner bar",
+      "BannerBarText":  "Visual element used to highlight the banner’s content.",
+      "TitleHeading":  "Title text",
+      "TitleText":  "The title of the banner.",
+      "DescriptionHeading":  "Description text",
+      "DescriptionText":  "The banner’s description or message.",
+      "CloseBtnHeading":  "Close button",
+      "CloseBtnText":  "Button that dismisses the banner.",
+      "CTAGroupHeading":  "CTA group",
+      "CTAGroupText":  "Group of actions that the user can interact with related to the banner content."
+    },
+    "Specs": {
+      "ImgAlt": "Specs displaying the padding and layer measurements within the banner component. Specs are described in the paragraph below.",
+      "Text": "The small banner with fluid layout has 12 pixels of vertical padding and 12 pixels of horizontal padding inside the banner. The icon has a 20-pixel width and height. There are 12 pixels between the icon and banner-content. The layout of the content within the banner adjusts based on the container’s width. The height of the banner si fluid and adjusts based on the amount of content within the banner. The large banner with fluid layout has 16 pixels of vertical padding and 16 pixels of horizontal padding inside the button. The icon has a 24-pixel width and height. There are 12 pixels between the icon and label. The layout of the content within the banner adjusts based on the container’s width. The height of the banner is fluid and adjusts based on the amount of content within the banner."
```

Comment:
Needs to add `Alt text description:` in the beginning
Same for French

Reply:
Fixed